app/controllers/pages_controller.rb

```diff
@@ -2,9 +2,13 @@
 
 class PagesController < ApplicationController
   def view_issues
-    # If our location.js has pulled a location, use it to find Stops
-    if (params[:lat]) && (params[:long])
-      @nearby_stops = Stop.includes(:lines).within(0.2, origin: [params[:lat], params[:long]])
+	# If rails is not prod, just take the top 50 stops.
+	# There is a known issue with SQLite and the geolocation gem
+	if (Rails.env != "production")
```

Comment:
There's already a nice function you can use for this - `Rails.env.production?`. See [env in the Rails docs](https://apidock.com/rails/Rails/env/class).

Reply:
Looks better, thanks for the tip